linux_os/guide/system/software/integrity/crypto/ssh_client_rekey_limit/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Configure session renegotiation for SSH client'
+
+description: |-
+    The <tt>RekeyLimit</tt> parameter specifies how often
+    the session key is renegotiated, both in terms of
+    amount of data that may be transmitted and the time
+    elapsed. To decrease the default limits, put line
+    <tt>RekeyLimit {{{ sub_var_value("var_ssh_client_rekey_limit_size") }}} {{{ sub_var_value("var_ssh_client_rekey_limit_time") }}}</tt> to file <tt>/etc/ssh/ssh_config.d/02-rekey-limit.conf</tt>.
+
+rationale: |-
+    By decreasing the limit based on the amount of data and enabling
+    time-based limit, effects of potential attacks against
+    encryption keys are limited.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: 82880-6
+
+references:
+    ospp: FCS_SSHS_EXT.1
+
+ocil_clause: 'it is commented out or is not set'
+
+ocil: |-
+    To check if RekeyLimit is set correctly, run the
+    following command:
+    <pre>$ sudo grep RekeyLimit /etc/ssh/ssh_config.d/02-rekey-limit.conf</pre>
```

Comment:
This is nice, but what happens when someone also adds it to /etc/ssh/sshd_config ?

Reply:
As per comments in the /etc/ssh/ssh_config file, the correct way of modification of system-wide ssh client configuration is creating files in /etc/ssh/ssh_config.d. If someone decides to do it othervise, I suppose they know what they are doing. Current rule ensures that the RekeyLimit will be configured by default as desired on default rhel8 system. There are many ways how to circumvent this; by using command line options when running the client, by creating a custom user config file... we can't check all of these. I chose this method because we already use it in hardening ssh client crypto-policy.

Reply:
@vojtapolasek  But that's for the crypto policy, this is more like site setting. In the analogy rule `sshd_rekey_limit` for sshd we check `/etc/ssg/sshd_config` instead of the `.d` directory.

Reply:
This PR needs to have both `/etc/ssh/ssh_config` and `/etc/sshd/ssh_config.d` handled; otherwise, it is considered incomplete and part of creating compliance content is handling both.